src/main/java/com/aws/greengrass/easysetup/DeviceProvisioningHelper.java

```diff
@@ -108,6 +110,8 @@
     private static final String IOT_ROLE_POLICY_NAME_PREFIX = "GreengrassTESCertificatePolicy";
     private static final String GREENGRASS_CLI_COMPONENT_NAME = "aws.greengrass.Cli";
     private static final String INITIAL_DEPLOYMENT_NAME_FORMAT = "Deployment for %s";
+    private static final String IAM_POLICY_ARN_FORMAT = "arn:%s:iam::%s:policy/%s";
+    private static final String MANAGED_IAM_POLICY_ARN_FORMAT = "arn:%s:iam::aws:policy/%s";
```

Comment:
Do we have a managed policy?

Reply:
We don't but since the arn is used to make a get request, it should look for it in both categories, this bit changed a bit now though after switching to the Arn builder